app/presenters/queries/content_item_presenter.rb

```diff
@@ -0,0 +1,24 @@
+module Presenters
+  module Queries
+    class ContentItemPresenter
+      def self.present(content_item)
+        version = Version.find_by(target: content_item)
+        new(content_item, version).present
+      end
+
+      def initialize(content_item, version)
+        self.content_item = content_item
+        self.version = version
+      end
+
+      def present
+        content_item.as_json.deep_symbolize_keys
+          .merge(version: version.number)
```

Comment:
I've discussed this with @tuzz to be certain this should not happen. Both GET commands do a `find_by(content_id: content_id)` and respond with a `CommandError` in the absence of a record.
We enforce version population on the PUT endpoints so retrieved records should all have a version. From what I remember of the version work, legacy data has been migrated to have a version too.